pkg/azure/plugin.go

```diff
@@ -17,8 +22,21 @@ func New() *Plugin {
 	}
 }
 
-func GetAvailableImplementations() map[string]func() plugin.Plugin {
-	return map[string]func() plugin.Plugin{
-		blob.PluginKey: blob.NewPlugin,
+func (p *Plugin) LoadConfig() error {
+	reader := bufio.NewReader(p.In)
+	b, err := ioutil.ReadAll(reader)
+	if err != nil {
+		return errors.Wrap(err, "could not read stdin")
+	}
+
+	if len(b) == 0 {
+		return nil
 	}
+
+	err = json.Unmarshal(b, &p.Config)
+	if err != nil {
+		return errors.Wrapf(err, "error unmarshaling stdin %q as azure.Config", string(b))
```

Comment:
`azure.Config` may need updating depending on what we settle on for pkg/import/struct name.  Or, perhaps we remove mention of the internal struct name itself... or generify?  (just made that one up.)

Reply:
Go's push to not repeat yourself (stuttering) causes more problems IMO. I'm going to try `azureconfig.Config` and see how that works.